core/src/main/scala/com/broadinstitute/dsp/DbReaderImplicits.scala

```diff
@@ -18,5 +18,10 @@ object DbReaderImplicits {
   implicit val gcsBucketNameGet: Get[GcsBucketName] = Get[String].map(GcsBucketName)
   implicit val locationGet: Get[Location] = Get[String].map(Location)
   implicit val kubernetesClusterNameGet: Get[KubernetesClusterName] = Get[String].map(KubernetesClusterName)
-  implicit val diskNameGet: Get[DiskName] = Get[String].map(DiskName)
+  implicit val diskNameMeta: Meta[DiskName] = Meta[String].imap(DiskName)(_.value)
+  implicit val zoneNameMeta: Meta[ZoneName] = Meta[String].imap(ZoneName)(_.value)
+  implicit val googleProjectMeta: Meta[GoogleProject] = Meta[String].imap(GoogleProject)(_.value)
+  implicit val k8sClusterNameMeta: Meta[KubernetesClusterName] = Meta[String].imap(KubernetesClusterName)(_.value)
+  implicit val locationMeta: Meta[Location] = Meta[String].imap(Location)(_.value)
+  implicit val regionNameMeta: Meta[RegionName] = Meta[String].imap(RegionName)(_.value)
```

Comment:
similar to `MappedColumnType` in slick...Defines how we want to map MySQL data type to custom types when we read data from MySQL and writing date to MySQL. `Meta` defines both ways. `Put` and `Get` defines one direction.

For case classes, doobie will be able to figure it out automatically so that we don't need to define something similar to slick's `def *` function